sample/Sample.hx

```diff
@@ -1,14 +1,10 @@
 import SampleModule.Point;
-import SampleModule.Init as SampleModuleInit;
+import SampleModule.Import;
 
 class Sample {
 
 	public static function main() {
-		SampleModuleInit.init(startApp);
```

Comment:
This shouldn't be removed as it is required for the JavaScript compilation and it doesn't break the HL compilation. I'm pretty sure that neither `Sample.hx` nor `SampleModule.hx` need to be changed. Did you have a problem with it? I'm able to build the sample for JS and HL without changing anything except the Makefile.

Reply:
Hmm...
In my environment (WSL, haxe 4.0.0-rc.1+1fdd3d5, hxcpp 4.0.8), `make hl` failed with this error:
```
haxe sample.hxml
Sample.hx:2: characters 8-20 : Module SampleModule does not define type SampleModule
```
I may do something wrong.

Reply:
Try running `haxelib dev webidl .` in the root of the webidl git repo before running make in the sample dir.

Reply:
It works!
Thank you, and sorry for bothering you.

Reply:
No problem. :slightly_smiling_face: You're actually the second user to run into that ( actually maybe the third if I count myself :wink: ), I'll go ahead and add a note to the readme about that.